core/api/src/domain/quiz/config.ts

```diff
@@ -11,10 +11,10 @@ export const QuizzesValue: Record<QuizQuestionId, Satoshis> = {
   moneyEvolution: 2 as Satoshis,
   whyStonesShellGold: 3 as Satoshis,
   moneyIsImportant: 3 as Satoshis,
-  moneyImportantGovernement: 4 as Satoshis,
+  moneyImportantGovernment: 4 as Satoshis,
   WhatIsFiat: 2 as Satoshis,
   whyCareAboutFiatMoney: 2 as Satoshis,
-  GovernementCanPrintMoney: 2 as Satoshis,
+  GovernmentCanPrintMoney: 2 as Satoshis,
```

Comment:
So we also leave this as is ? @nicolasburtey

Reply:
yes